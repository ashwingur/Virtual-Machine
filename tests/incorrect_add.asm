FUNC LABEL 0
    PRINT VAL 6
    MOV STK A VAL 2
    ADD STK A VAL 7
    RET