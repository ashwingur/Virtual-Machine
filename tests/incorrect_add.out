6
The arguments for ADD must be of register type.
